singleuser/DockerfileIDE

```diff
@@ -0,0 +1,73 @@
+ARG JUPYTERHUB_VERSION
```

Comment:
if you name the file IDE.Dockerfile it will enable syntax highlight in IDE and Github